articles/quickstart/webapp/java-spring-boot/01-login.md

```diff
@@ -71,64 +77,56 @@ If you are using Maven:
     </dependency>
     <dependency>
         <groupId>org.thymeleaf.extras</groupId>
-        <artifactId>thymeleaf-extras-springsecurity5</artifactId>
+        <artifactId>thymeleaf-extras-springsecurity6</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>nz.net.ultraq.thymeleaf</groupId>
+        <artifactId>thymeleaf-layout-dialect</artifactId>
     </dependency>
 </dependencies>
 ```
 
-:::note
-The Spring Security 5.4.0 release will include [a fix](https://github.com/spring-projects/spring-security/pull/8357) to validate the issuer claim of the ID token. You should update to this release when available.
-:::
-
 ### Configure Spring Security
 
-Spring Security makes it easy to configure your application for authentication with OIDC providers such as Auth0. In your application's configuration, configure the OAuth2 client and provider. The sample below uses an `application.yml` file, though you can also use properties files or any of the other [supported externalization mechanisms](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config).
+The Okta Spring Boot Starter makes it easy to configure your application with Auth0. The sample below uses an `application.yml` file, though you can also use properties files or any of the other [supported externalization mechanisms](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config).
 
-:::note
-Spring Security will use the `issuer-uri` property value to retrieve all the information necessary to enable login and validation of the ID token at runtime.
-
-[Additional property mappings](https://docs.spring.io/spring-security/site/docs/current/reference/html5/#oauth2login-boot-property-mappings) are available for further customization, if required.
-:::
 
 ```yaml
 # src/main/resources/application.yml
-spring:
-  security:
-    oauth2:
-      client:
-        registration:
-          auth0:
-            client-id: ${account.clientId}
-            client-secret: YOUR_CLIENT_SECRET
-            scope:
-              - openid
-              - profile
-              - email
-        provider:
-          auth0:
-            # trailing slash is important!
-            issuer-uri: https://${account.namespace}/
+okta:
+  oauth2:
+    issuer: https://${account.namespace}/
+    client-id: ${account.clientId}
```

Comment:
Are these variables that are replaced? Otherwise, the syntax looks funny.

Reply:
Yes `account.namespace` becomes the domain, and `account.clientId` the corresponding clientId based on the selected application when logged in in the docs platform. When not logged in, these gets replaced by placeholders.

Reply:
@mraible yes, they are replaced when a user logs in and downloads the sample.